clients/flash/common-library/src/main/actionscript/org/bigbluebutton/lib/settings/views/lock/LockSettingsViewBase.as

```diff
@@ -0,0 +1,177 @@
+			addElement(_lockTitle);
+			
+			// Lock group
+			var lockGroup:HGroup = new HGroup();
+			lockGroup.percentWidth = 100;
+			lockGroup.verticalAlign = VerticalAlign.MIDDLE;
+			addElement(lockGroup);
+			
+			var lockLabel:Label = new Label();
+			lockLabel.text = "Lock all participants";
+			lockLabel.percentWidth = 100;
+			lockGroup.addElement(lockLabel);
+			
+			_lockToggle = new toggleButtonClass();
+			lockGroup.addElement(_lockToggle);
+			
+			// Webcam group
+			var webcamGroup:HGroup = new HGroup();
+			webcamGroup.percentWidth = 100;
+			webcamGroup.verticalAlign = VerticalAlign.MIDDLE;
+			addElement(webcamGroup);
+			
+			var webcamLabel:Label = new Label();
+			webcamLabel.text = "Webcam";
+			webcamLabel.percentWidth = 100;
+			webcamGroup.addElement(webcamLabel);
+			
+			_webcamCheckbox = new CheckBox();
+			webcamGroup.addElement(_webcamCheckbox);
+			
+			// Microphone group
+			var microphoneGroup:HGroup = new HGroup();
+			microphoneGroup.percentWidth = 100;
+			microphoneGroup.verticalAlign = VerticalAlign.MIDDLE;
+			addElement(microphoneGroup);
+			
+			var microphoneLabel:Label = new Label();
+			microphoneLabel.text = "Microphone";
+			microphoneLabel.percentWidth = 100;
+			microphoneGroup.addElement(microphoneLabel);
+			
+			_microphoneCheckbox = new CheckBox();
+			microphoneGroup.addElement(_microphoneCheckbox);
+			
+			// Public chat group
+			var publicChatGroup:HGroup = new HGroup();
+			publicChatGroup.percentWidth = 100;
+			publicChatGroup.verticalAlign = VerticalAlign.MIDDLE;
+			addElement(publicChatGroup);
+			
+			var publicChatLabel:Label = new Label();
+			publicChatLabel.text = "Public chat";
+			publicChatLabel.percentWidth = 100;
+			publicChatGroup.addElement(publicChatLabel);
+			
+			_publicChatCheckbox = new CheckBox();
+			publicChatGroup.addElement(_publicChatCheckbox);
+			
+			// Private chat group
+			var privateChatGroup:HGroup = new HGroup();
+			privateChatGroup.percentWidth = 100;
+			privateChatGroup.verticalAlign = VerticalAlign.MIDDLE;
+			addElement(privateChatGroup);
+			
+			var privateChatLabel:Label = new Label();
+			privateChatLabel.text = "Private chat";
+			privateChatLabel.percentWidth = 100;
+			privateChatGroup.addElement(privateChatLabel);
+			
+			_privateChatCheckbox = new CheckBox();
+			privateChatGroup.addElement(_privateChatCheckbox);
+			
+			// Layout group
+			var layoutGroup:HGroup = new HGroup();
+			layoutGroup.percentWidth = 100;
+			layoutGroup.verticalAlign = VerticalAlign.MIDDLE;
+			addElement(layoutGroup);
+			
+			var layoutLabel:Label = new Label();
+			layoutLabel.text = "Layout";
+			layoutLabel.percentWidth = 100;
+			layoutGroup.addElement(layoutLabel);
+			
+			_layoutCheckbox = new CheckBox();
+			layoutGroup.addElement(_layoutCheckbox);
+		}
+		
+		override protected function updateDisplayList(w:Number, h:Number):void {
+			super.updateDisplayList(w, h);
+			
+			_muteToggle.parent["padding"] = getStyle("padding");

```

Comment:
The `padding` needs to be set for the container of the Checkbox, we may have more than one component inside it to align.